Source/DataSources/Entity.js

```diff
@@ -581,9 +581,14 @@ define([
     var orientationScratch = new Quaternion();
 
     /**
-     * @private
+     * Computes the model matrix for the entity's transform at specified time.
+     *
```

Comment:
Expand on this documentation to indicate what happens if the entity has no position or no orientation.